beta/project-griffon/using-project-griffon/custom-validation-and-project-griffon.md

```diff
@@ -10,16 +10,16 @@ To use this view, complete the following steps:
 
 1. [Setup Project Griffon](../set-up-project-griffon.md)
 2. [Create](./#creating-sessions) and [connect](./#connecting-to-a-session) to a Project Griffon session
-3. In the **Home** view, select **Custom Validation**
+3. In the **Home** view, select **Validation Scripts** (to be deprecated for **Validation Summary**)
 
-![Custom Validation](../../../.gitbook/assets/screen-shot-2020-02-12-at-9.17.25-am.png)
+![Custom Validation](../../../.gitbook/assets/Screen Shot 2022-02-24 at 11.41.55 AM.png)
```

Comment:
I recommend using a meaningful file name.

Reply:
I saw other screenshots in there and got lazy. Updated.